docs/DEV_ENVIRONMENT.md

```diff
@@ -66,5 +66,16 @@
    CC=/opt/homebrew/opt/llvm/bin/clang AR=/opt/homebrew/opt/llvm/bin/llvm-ar wasm-pack test --firefox --headless mm2src/mm2_main
    ```
    Please note `CC` and `AR` must be specified in the same line as `wasm-pack test mm2src/mm2_main`.
+#### Running WASM tests with Cargo</br>   
```

Comment:
To me, the way this looks feels too much like "you can run tests with cargo without needing to setup things above".

Reply:
hmmm, I think this is mostly useful when you want to run a single or some specific `wasm` tests alone.

Reply:
I understand, but keep in mind that the command can also be used to run the full test just like `wasm-pack`..hence why I used the title as it's not just limited to running single/some tests

Reply:
I know but I don't think anyone will think cargo can only do single test. Command explicitly contains flags that everyone can understand. I want to avoid the confusion of "there is an alternative way of running wasm tests that don't require prequests mentioned above".

Reply:
ok got it. amended now

Reply:
Can we run single test using `wasm-pack`? If we can, we should use `wasm-pack` for "example of running single test", and find better title for `cargo` one probably.

Reply:
personally, I'm not sure if `wasm-pack` can perform same action tho